README.md

```diff
@@ -135,3 +135,17 @@ Stop k3s after manual testing:
 ```sh
 mvn k3s:rm
 ```
+
+## Contributing
+
+In order to contribute to the project:
+
+* create a fork and apply your changes
+* open a PullRequest and apply at least one of the following labels:
```

Comment:
can this be done by a maintainer? I think the contributor not always know it.

Reply:
The check_pr action will add a comment to the PR in case none is applied. But it can also be done by the maintainers. Both is supported.